R/validator.R

```diff
@@ -267,6 +269,10 @@ InputValidator <- R6::R6Class("InputValidator", cloneable = FALSE,
       result
     },
     # indent is character() if logging, FALSE if not
+    #' @description Generate validation logs which are visible in the R Console
+    #'   if the option `shinyvalidate.verbose` is `TRUE`.
+    #' @param indent The indentation string to be used when printing log
+    #'   entries.
```

Comment:
Oh, right, this needs to be made private.

Reply:
Did you want to fix that up and merge or should I make the change?